### ISSUE TITLE
Rails 6 support?

### ISSUE THREAD

**Issue by artempartos** [ianheggie/health_check#94]

**Comment by puneetpandey** [comment]
+1

Any updates on Rails6?

We have recently upgraded our staging environment to Rails6 but after upgrading `health_check` url started throwing errors: `: (92) HTTP/2 stream 1 was not closed cleanly: INTERNAL_ERROR (err 2)`. Our pre-prod and prod environments (with Rails-5.2.3) are working as expected

**Comment by shlima** [comment]
@artempartos @puneetpandey you can check an [alternative](https://github.com/shlima/health_bit)

**Comment by AxelTheGerman** [comment]
@shlima thank you for stepping in - I also thought this gem might be a bit too much (and not maintained 😞 ). Had a look at `health_bit` and looks nice and simple, will give it a try the next few days!

**Comment by ianheggie** [comment]
> @shlima thank you for stepping in - I also thought this gem might be a bit too much (and not maintained disappointed ). Had a look at `health_bit` and looks nice and simple, will give it a try the next few days!

How did you find health_bit? I put a link in the README to it and sent a note to the author recommending CI testing like trvis, but it does look good at a quick glance.

PS Rails 6 support has passed testing, am adding some extra PR's before the next release

**Comment by ianheggie** [comment]
Fixed in version 3.1.0 released just now

**Comment by AxelTheGerman** [comment]
Actually haven't gotten around it yet. I previously wrote a very simple endpoint myself and guess didn't need much more since

```ruby
class Status::HealthCheckController < ApplicationController
  skip_before_action :require_valid_user!
  skip_before_action :verify_authenticity_token

  OK = 'OK'
  ERROR = 'ERROR'

  def simple
    render plain: OK
  end

  def db
    ok = ActiveRecord::Base.connection.current_database.present?
    render plain: ok ? OK : ERROR, status: ok ? :ok : :internal_server_error
  rescue
    render plain: ERROR, status: :internal_server_error
  end
end
```
